styleguide/derek/templates/thought-leadership-overview/_partials/about-author.ejs

```diff
@@ -0,0 +1,17 @@
+<div class="rsTl-author-section">
+  <h2 class="rsTl-author-title">About the Author</h2>
+  <div class="rsTl-author-sectionInner">
+    <div class="rsTl-author-photo" style="background-image:url('imgs/derek/speaker-thumb.jpg');"></div>
+    <div class="rsTl-author-text">
+      <span class="rsTl-author-label">CEO | Rackspace</span>
+      <span class="rsTl-author-name">Derek Zoolander</span>
+      <p>
+        Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium
+        doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore
+        veritatis et quasi architecto beatae vitae dicta sunt explicabo.
+      </p>
+      <a class="rsTl-social-twitter" href="https://twitter.com" target="_blank"></a>
+      <a class="rsTl-social-linkedin" href="https://linkedin.com" target="_blank"></a>
```

Comment:
Can you add `rel="noopener noreferrer"` to all `target="_blank"` links per https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/